Require.sublime-settings

```diff
@@ -40,6 +40,9 @@
     // Use 'single' or "double" quotes
     "quotes": "single",
 
+    // Use 'var' or 'let' variable declarations
+    "var": "let",
```

Comment:
I think we should still default to var for this setting, but I guess it doesn't really matter that much. In most cases, the scope won't be affected by using let vs var in this context I believe.